### ISSUE TITLE
Wrong color for hint text

### ISSUE THREAD

**Issue by calebegg** [angular/material#3998]
Hint text in <md-input-container> and friends is too light; the contrast ratio for hint text is only 1.88. The Material spec requires a ratio of at least 4.5. Hint text should be changed to 54% alpha instead of the current 26%.

http://www.google.com/design/spec/style/typography.html#typography-other-typographic-guidelines

"Text should maintain a minimum contrast ratio of at least 4.5:1"

**Comment by bluebirdtech** [comment]
+1

I'm seeing "three shades of grey".. when looking at the inputs: https://material.angularjs.org/HEAD/#/demo/material.components.input

 - label non-floating state: BDBDBD (lightest - too light in fact)
 - label floating state: 757575 (darkest)
 - input placeholder: A9A9A9 (medium)

It would be great if these were all consistent.  Inputs on the material design spec seem to be about A0A0A0.  https://www.google.co.uk/design/spec/components/text-fields.html#text-fields-single-line-text-field

Of course these are all shown on an off-white background: FAFAFA.

**Comment by salahkamel** [comment]
Do we have an idea why this is postponed post 1.0? Is it more than just adjusting the colors in the CSS? I might be missing something.

Thanks!
-Salah

**Comment by EladBezalel** [comment]
A PR is very welcome!

**Comment by enderdickerson** [comment]
What's the status on this?

**Comment by Splaktar** [comment]
In `1.2.0`, there are significant improvements to contrasts and palettes. PR https://github.com/angular/material/pull/11953 builds on this and should resolve this issue as well.
